### PR TITLE
Asset Centric Wallet - OperationsHistorySection added to the Wallet page

### PR DESCRIPTION
**Pull request by cgrellard-ledger**
### 📝 Description

OperationsHistorySection added to the Wallet page

### ❓ Context

- **Impacted projects**: `ledger-live-mobile` 
- **Linked resource(s)**: [LIVE-2799] 

### ✅ Checklist

- [ ] **Test coverage** 
- [x] **Atomic delivery** 
- [x] **No breaking changes** 

### 📸 Demo

Screenshot_2022-08-02-11-09-21-05_99c1b3bb631177f8a9e1379ae5b12ffe

### 🚀 Expectations to reach

_Please make sure you follow these [**Important Steps**](https://github.com/LedgerHQ/ledger-live/blob/develop/CONTRIBUTING.md#important-steps)._

_Pull Requests must pass the CI and be internally validated in order to be merged._

[LIVE-2799]: https://ledgerhq.atlassian.net/browse/LIVE-2799?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

**Review by Justkant — COMMENTED** [review]
You're missing a changeset.

Also while looking at the screenshot provided in the description. I find the "Transaction History" header a bit too close to the previous component just over it.

File: apps/ledger-live-mobile/src/screens/Portfolio/index.tsx [hunk 1]
```diff
@@ -256,15 +254,7 @@ function PortfolioScreen({ navigation }: Props) {
         : []),
       ...(showAssets
         ? [
-            <Box background={colors.background.main}>
```
Comment (nparigi-ledger) [inline apps/ledger-live-mobile/src/screens/Portfolio/index.tsx:259]: Shouldn't we keep the same format as before ?
SectionTitle outside of the section.
And, like i see on the screen, if the section title css have changed, just change it to the new one ? 

Something like this : 

`
 <Box>
<SectionContainer px={0} mb={8}>
                <SectionTitle
                  title={t("analytics.operations.title")}
                  containerProps={{ mx: 6 }}
                />
                <OperationsHistorySection accounts={assetsToDisplay} />,
              </SectionContainer>
            </Box>,
`

**Comment by cgrellard-ledger** [comment]
> You're missing a changeset.
> 
> Also while looking at the screenshot provided in the description. I find the "Transaction History" header a bit too close to the previous component just over it.

I feel like it will make more sense to just add a changeset for the whole feature rather than for every small sub task
The component above will be removed in another subtask for this feature so it doesn't really matter that the two are really close for now

**Comment by Justkant** [comment]
> > You're missing a changeset.
> > Also while looking at the screenshot provided in the description. I find the "Transaction History" header a bit too close to the previous component just over it.
> 
> I feel like it will make more sense to just add a changeset for the whole feature rather than for every small sub task The component above will be removed in another subtask for this feature so it doesn't really matter that the two are really close for now

You're right I didn't see the base branch wasn't `develop`

**Review by nparigi-ledger — APPROVED** [review]

**Pull request merged**